scrapi/harvesters/neurovault.py

```diff
@@ -0,0 +1,79 @@
+"""
+NeuroVault (neurovault.org) harvester of public collections for the
+SHARE Notification Service.
+
+Example API query: http://neurovault.org/api/collections/?format=json
+"""
+
+from __future__ import unicode_literals
+
+import json
+import logging
+# from datetime import date, timedelta, datetime
+import re
+from scrapi import requests
+# from scrapi import settings
+from scrapi.base import JSONHarvester
+from scrapi.linter.document import RawDocument
+from scrapi.base.helpers import (build_properties, datetime_formatter,
+                                 default_name_parser)
+
+logger = logging.getLogger(__name__)
+
+
+def process_contributors(authors):
+
+    if authors is None:
+        return []
+    authors = re.split(',\s|\sand\s', authors)
+    return default_name_parser([a for a in authors])
+
+
+class NeuroVaultHarvester(JSONHarvester):
+    short_name = 'neurovault'
+    long_name = 'NeuroVault.org'
+    url = 'http://www.neurovault.org/'
+    count = 0
+
+    @property
+    def schema(self):
+        return {
+            'contributors': ('/authors', process_contributors),
+            'uris': {
+                'canonicalUri': '/url',
+            },
+            'title': '/name',
+            'providerUpdatedDateTime': ('/modify_date', datetime_formatter),
+            'description': '/description',
+            'otherProperties': build_properties(
+                ('owner_name', '/owner_name'),
+                ('doi', '/DOI'),
```

Comment:
This is actually a great link to put into `descriptorUris`, I think, which are supposed to point to descriptions of the research object (ie, articles, abstracts, landing pages, etc). I'd recommend something like this:
```python
'uris': {
    ...,
    'descriptorUris': ('/DOI', lambda x: ['http://dx.doi.org/{}'.format(x)]),
}
```

In the search interface we only expose the `canonicalUri`, so the vast majority of people will only ever see that. These other fields are mostly for our own benefit, as we can use the additional identifiers for disambiguation.